xc-tool-ts/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import {MainnetConfig, TestnetConfig, AttesterConfigState, LocalDevnetConfig, getEvmDataSources, getAttesterConfig, Config} from "./config";
+import {PublicKey} from "@solana/web3.js";
+const DEFAULTS: Config = {
```

Comment:
(suggest moving this to a json or yaml configuration file at some point in the future)

Reply:
See my response to the mainnet/testnet divide nit.